tox.ini

```diff
@@ -5,8 +5,11 @@ max-line-length = 100
 skipsdist = True
 
 [testenv:unit]
+setenv =
+    PYTHONBREAKPOINT=ipdb.set_trace
```

Comment:
Could these `tox.ini` changes get moved to a separate commit/PR?

Reply:
Sorry, left that in from debugging.